service/src/tests/migration.rs

```diff
@@ -0,0 +1,309 @@
+use std::{collections::BTreeSet, io::Cursor, str::FromStr};
+
+use anyhow::{anyhow, Result};
+use async_trait::async_trait;
+use ceramic_core::{DidDocument, EventId, Network, StreamId};
+use ceramic_event::unvalidated;
+use cid::Cid;
+use futures::{stream::BoxStream, Stream, StreamExt as _, TryStreamExt as _};
+use ipld_core::{codec::Codec, ipld, ipld::Ipld};
+use iroh_car::CarReader;
+use multihash_codetable::{Code, MultihashDigest};
+use rand::{thread_rng, Rng, RngCore};
+use recon::Key;
+use serde_ipld_dagcbor::codec::DagCborCodec;
+use test_log::test;
+
+use crate::{Block, BoxedBlock, CeramicEventService};
+
+struct InMemBlock {
+    cid: Cid,
+    data: Vec<u8>,
+}
+#[async_trait]
+impl Block for InMemBlock {
+    fn cid(&self) -> Cid {
+        self.cid
+    }
+    async fn data(&self) -> Result<Vec<u8>> {
+        Ok(self.data.clone())
+    }
+}
+async fn blocks_from_cars(cars: Vec<Vec<u8>>) -> impl Stream<Item = Result<BoxedBlock>> {
+    let mut stream: Option<BoxStream<Result<BoxedBlock>>> = None;
+    for car in cars {
+        let reader = CarReader::new(Cursor::new(car)).await.unwrap();
+        let s = reader
+            .stream()
+            .map_err(|err| anyhow!("{err}"))
+            .map(|block| block.map(|(cid, data)| Box::new(InMemBlock { cid, data }) as BoxedBlock));
+        if let Some(prev_stream) = stream.take() {
+            stream = Some(prev_stream.chain(s).boxed());
+        } else {
+            stream = Some(s.boxed());
+        }
+    }
+    //TODO return empty stream
```

Comment:
What's the reason to do this? Seems like it's working for the tests now?

Reply:
Yeah, this was more for completeness than anything. But no tests need this code path so I'll remove the todo so its clear its not broken.